src/test/java/scmspain/karyon/restrouter/KaryonRestRouterTest.java

```diff
@@ -246,4 +251,33 @@ public void testEndpointWithQueryParamsReturnIntegerObs() throws Exception {
     Assert.assertEquals("{\"value\":\"1\"}", body);
   }
 
+  @Test
+  public void testHystrixStreamEndpointReturnsOk() throws Exception {
+
+    // This request is wrapper in a Hystrix command because, since /hystrix.stream does not send
+    // response's headers until it send the first byte, and to do so need some metrics to exists,
+    // it needs any Hystrix command to be executed to start the stream.
+    new HystrixObservableCommand<HttpClientResponse<ByteBuf>>(HystrixCommandGroupKey.Factory.asKey("request")) {
+      @Override
+      protected Observable<HttpClientResponse<ByteBuf>> construct() {
+        return RxNetty.createHttpClient("localhost", AppServer.KaryonRestRouterModuleImpl.DEFAULT_PORT)
+          .submit(HttpClientRequest.createGet("/example/1"));
+      }
+    }.construct().map(this::callToHystrixStreamEndpoint);
+  }
+
+  private String callToHystrixStreamEndpoint(HttpClientResponse<ByteBuf> any) throws RuntimeException {
```

Comment:
I deleted the test as last chance before commit suicide xD
Just read carefully tests' documentation: https://github.com/scm-spain/karyon-rest-router/pull/25/files#diff-affe0e4f87e820c2f801be43cbddf613L68